lib/node_modules/@stdlib/lapack/base/dlange/lib/base.js

```diff
@@ -0,0 +1,367 @@
+				iw += strideWork;
+				ia2 += strideA2;
+			}
+			ia1 += strideA1;
+		}
+
+		value = 0.0;
+
+		iw = offsetWork;
+		for ( i = 0; i < N; i++ ) {
+			temp = work[ iw ];
+			if ( value < temp || isnan( temp ) ) {
+				value = temp;
+			}
+			iw += strideWork;
+		}
+	} else {
+		value = 0.0;
+		ia1 = offsetA;
+		for ( j = 0; j < N; j++ ) {
+			sum = dasum( M, A, strideA1, ia1 );
+			if ( value < sum || isnan( sum ) ) {
+				value = sum;
+			}
+			ia1 += strideA2;
+		}
+	}
+
+	return value;
+}
+
+/**
+* Returns the absolute value of the maximum element of a real matrix `A`.
+*
+* @private
+* @param {NonNegativeInteger} M - number of rows in `A`
+* @param {NonNegativeInteger} N - number of columns in `A`
+* @param {Float64Array} A - input array
+* @param {integer} strideA1 - stride of the first dimension of `A`
+* @param {integer} strideA2 - stride of the second dimension of `A`
+* @param {NonNegativeInteger} offsetA - starting index of `A`
+* @returns {number} result
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var A = new Float64Array( [ 1.0, 4.0, 7.0, 10.0, 2.0, 5.0, 8.0, 11.0, 3.0, 6.0, 9.0, 12.0 ] );
+*
+* var out = maxAbs( 3, 4, A, 4, 1, 0 );
+* // returns 12.0
+*/
+function maxAbs( M, N, A, strideA1, strideA2, offsetA ) {
+	var value;
+	var temp;
+	var da0;
+	var sa1;
+	var sa2;
+	var da1;
+	var ia;
+	var S1;
+	var S2;
+	var i;
+	var j;
+
+	value = 0.0;
+
+	if ( isRowMajor( [ strideA1, strideA2 ] ) ) {
+		S1 = M;
+		S2 = N;
+		sa1 = strideA1;
+		sa2 = strideA2;
+	} else {
+		S1 = N;
+		S2 = M;
+		sa1 = strideA2;
+		sa2 = strideA1;
+	}
+
+	da0 = sa2;
+	da1 = sa1 - (S2 * sa2);
+	ia = offsetA;
+
+	for ( i = 0; i < S1; i++ ) {
+		for ( j = 0; j < S2; j++ ) {
+			temp = A[ ia ];
+			if ( value < temp || isnan( temp ) ) {
+				value = temp;
+			}
+			ia += da0;
+		}
+		ia += da1;
+	}
+	return value;
+}
+
+/**
+* Returns the value of the infinity norm of a real matrix `A`.
+*
+* @private
+* @param {NonNegativeInteger} M - number of rows in `A`
+* @param {NonNegativeInteger} N - number of columns in `A`
+* @param {Float64Array} A - input array
+* @param {integer} strideA1 - stride of the first dimension of `A`
+* @param {integer} strideA2 - stride of the second dimension of `A`
+* @param {NonNegativeInteger} offsetA - starting index of `A`
+* @param {Float64Array} work - work array, should have `M` indexed elements if column-major layout is used
+* @param {integer} strideWork - stride length of `work`
+* @param {NonNegativeInteger} offsetWork - starting index of `work`
+* @returns {number} result
+*
+* @example
+* var Float64Array = require( '@stdlib/array/float64' );
+*
+* var A = new Float64Array( [ 1.0, 4.0, 7.0, 10.0, 2.0, 5.0, 8.0, 11.0, 3.0, 6.0, 9.0, 12.0 ] );
+* var work = new Float64Array( 3 );
+*
+* var out = infinityNorm( 3, 4, A, 4, 1, 0, work, 1, 0 );
+* // returns 30.0
+*/
+function infinityNorm( M, N, A, strideA1, strideA2, offsetA, work, strideWork, offsetWork ) { // eslint-disable-line max-len
```

Comment:
Is it me or is the `infinityNorm` just the `oneNorm` with `A` transposed? If so, can we just consolidate implementations into a single function and then have `oneNorm` and `infinityNorm` be light wrappers which swap the strides as necessary?